### DISCUSSION TITLE
[Question] Windows Installation

### DISCUSSION THREAD

**Discussion by phreaq** [arabcoders/watchstate#214] [General] [answered]
Does anyone have any documentation on how to run this in Windows, presumably using Docker Desktop, or something similar?

I'm not familiar with docker, and when trying to run 

`docker-compose pull && docker-compose up -d`

I get errors

1 error(s) decoding:

`* error decoding 'Volumes[0]': invalid spec: :/config:rw: empty section between colons`

I know I'm a fish out of water at the moment, but if there's any relevant guides out there, I'll read up on it, and go from there.

**Comment by arabcoders** [comment, marked as answer]
Hi

You have to replace the volume mount to a folder for example:

```yaml
volumes:
  - ${PWD}:/config
```

to 

```yaml
volumes:
  - ./data:/config
```

**Reply by phreaq** [reply]
perfect, thanks!
